### PR TITLE
add webrtc docs

### PR DESCRIPTION
**Pull request by yxlao**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/3506)

### CODE REVIEW

**Comment by yxlao** [comment]
Preview:
screencapture-file-home-yixing-repo-Open3D-docs-out-html-tutorial-visualization-web-visualizer-html-2021-05-27-06_29_19

**Review by ssheorey — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/intel-isl/open3d/3506)* status: 0 of 6 files reviewed, 9 unresolved discussions (waiting on @germanros1987 and @yxlao)

___
*[docs/tutorial/visualization/web_visualizer.rst, line 43 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiMYvl1riQtBKb_sFy:-MaiMYvm0wZLEZnWyJwK:b-9djbg7) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L43)):*
> ```reStructuredText
> Additional notes on compatibility:
> 
> - Web visualizer servers are not supported on the ARM64 platform.
> ```

ARM64 -> ARM (include old Raspberry Pi)
___
*[docs/tutorial/visualization/web_visualizer.rst, line 56 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiN24X1XcbbT1jBWMW:-MaiN24X1XcbbT1jBWMX:b12hia4) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L56)):*
> ```reStructuredText
> 
> In standalone mode, Open3D web visualizer server runs as a standalone
> application. It supports both C++ and Python. You'll only need make very minimal
> ```

need **to** make very minimal **changes**
___
*[docs/tutorial/visualization/web_visualizer.rst, line 126 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiOfeOCJIXxPDMxWfV:-MaiOfeOCJIXxPDMxWfW:bxx3hzw) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L126)):*
> ```reStructuredText
> enable HTTP traffic. Finally, we start the web visualizer server and visit the
> external IP address from the browser.
> ```

.. warning::

The web server uses ``http`` by default which is not encrypted and is suitable only for local access. If used for remote visualization, the web page will be visible and controllable by any one with network access. For sensitive or confidential data, please rebuild with `appropriate web server configuration and SSL certificates <https://github.com/civetweb/civetweb/blob/master/docs/OpenSSL.md>`__.
___
*[docs/tutorial/visualization/web_visualizer.rst, line 146 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiiXG4EaXOpivdUh8B:-MaiiXG4EaXOpivdUh8C:b3ud833) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L146)):*
> ```reStructuredText
> 
> Then, run the example in
> ``examples/python/visualization/jupyter_visualization.ipynb``.
> ```

.. warning:: 

For remote visualization, please use a secure Jupyter server as described `here <https://jupyter-notebook.readthedocs.io/en/stable/public_server.html>`__
___
*[docs/tutorial/visualization/web_visualizer.rst, line 151 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiZeY75KQlcFf3eo26:-MaiZeY75KQlcFf3eo27:b-wchjpv) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L151)):*
> ```
> avoid avoid 
> ```
___
*[docs/tutorial/visualization/web_visualizer.rst, line 165 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiZvvjC2Z8CTq57hty:-MaiZvvjC2Z8CTq57htz:bqylnnh) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L165)):*
> ```reStructuredText
> :::::::::::::::::::::::::::::::::
> 
> Open3D official Python wheels comes with Jupyter web visualizer support.
> ```

come with
___
*[docs/tutorial/visualization/web_visualizer.rst, line 193 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-Mai_HQ1ErJegpylFOqn:-Mai_HQ1ErJegpylFOqo:b-rzyi9o) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L193)):*
> ```
> failed
> ```
fail
___
*[docs/tutorial/visualization/web_visualizer.rst, line 226 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-Mai_i5L6J-AyC8t7YK7:-Mai_i5L6J-AyC8t7YK8:befciwr) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L226)):*
> ```
> TURN server (video traffic relay)
> ```
___
*[docs/tutorial/visualization/web_visualizer.rst, line 237 at r2](https://reviewable.io/reviews/intel-isl/open3d/3506#-MaiaE09BIDngc4dsI0Z:-MaiaE09BIDngc4dsI0_:bt5i7te) ([raw file](https://github.com/intel-isl/open3d/blob/476557e95ad0cc5adf5fcd8f7531f1aef32dd0c4/docs/tutorial/visualization/web_visualizer.rst#L237)):*
> ```reStructuredText
>     WEBRTC_STUN_SERVER="turn:user:password@my_tcp_turn_server.com:3478?transport=tcp"
>     # UDP and TCP (more than one TURN servers)
>     WEBRTC_STUN_SERVER="turn:user:password@my_turn_server.com:3478;turn:user:password@my_tcp_turn_server.com:3478?transport=tcp"
> ```

6. Advanced topic: Network issues
-------------------------------------------
You may face issues such as failure to connect, lag or visualization quality in specific network configurations. If the visualization window in the notebook or web page is blank, WebRTC could not establish a connection, likely due to your specific NAT or VPN configuration.

< Debugging ideas or webrtc tests to try here>
e.g. Test both client and server machines for Connectivity and Throughput at https://test.webrtc.org/

If you face control lag, try switching from TCP to UDP connection or switching to a closer TURN server.

If the visualization video has compression artifacts, you may be using a TURN server (video traffic relay) with insufficient bandwidth. Try disabling TURN servers or switching to a server with higher bandwidth.

**Review by ssheorey — CHANGES_REQUESTED** [review]
LGTM except for the security issues warning. The debugging section would be useful but its OK if we want to skip it for now.

File: docs/tutorial/visualization/web_visualizer.rst [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+Additional notes on compatibility:
+
+- On Ubuntu, web visualizer server runs on EGL. This is supported in most
+  machines as long as the it has graphics capabilities (either via integrated
+  graphics or discrete GPU). A discrete GPU will be required if the CPU does not
+  come with integrated graphics. For example, AWS cloud instances typically use
+  Intel XEON processors without integrated graphics. You'll need to add a
+  discrete GPU to the instance and install the graphics driver to enable the web
+  visualizer.
+- Web visualizer servers are not supported on the ARM6 platform.
```
Comment (ssheorey) [inline docs/tutorial/visualization/web_visualizer.rst:53]: ARM6 -> ARM

**Review by ssheorey — APPROVED** [review]

**Pull request merged**